src/services/signedTransaction.ts

```diff
@@ -112,13 +97,28 @@ export const handleSignedTx = async (
 ): Promise<void> => {
   if (!req.body.signedTx) throw new Error("No signedTx in body");
 
-  if (req.body.customNodeUrl) {
-    await submitToCustomURL(req, res);
-    return;
-  }
   if (config.get("usingQueueEndpoint") === "true") {
     await submitToQueue(req, res);
   } else {
     await submit(req, res);
   }
 };
+
+const blockfrostErrorMsg = (status: number) => {
```

Comment:
What's the benefit of this? Blockfrost already returns the error messages as part of its reponse

Reply:
This I added in case the error status is not 200, to handle the scenario when there will be no data, just to be explicit about the error.  

As something similar was done here - 
https://github.com/dcSpark/cardano-backend/blob/master/src/services/signedTransaction.ts#L42

Reply:
please follow the Blockfrost documentation about how to get the error message instead. If we duplicate the error messages, they will be wrong if Blockfrost ever changes their API

Reply:
Okay, refactored.